examples/.octopus/workflow/octopus.yaml

```diff
@@ -0,0 +1,48 @@
+---
+SpaceName: Default
+ProjectName: actions-test
+
+Process:
+  - Name: Debug
+    PackageRequirement: LetOctopusDecide
+    Condition: Success
+    StartTrigger: StartAfterPrevious
+    Actions:
+    - Name: Debug
+      ActionType: Octopus.Script
+      Notes:
+      IsDisabled: false
+      CanBeUsedForProjectVersioning: true
+      IsRequired: false
+      WorkerPoolVariable: WORKER_POOL
+      Properties:
+        Octopus.Action.RunOnServer: 'true'
+        Octopus.Action.Script.ScriptSource: Inline
+        Octopus.Action.Script.Syntax: Python
+        Octopus.Action.Script.ScriptBody: print("INCLUDE DEBUG SCRIPTS HERE")
+      Links: {}
+  - Name: TF Apply
+    PackageRequirement: LetOctopusDecide
+    Properties:
+      Octopus.Step.ConditionVariableExpression: "#{AWS_ACCESS_KEY_ID}"
+    Condition: Always
+    StartTrigger: StartAfterPrevious
+    Actions:
+    - Name: TF Apply
+      ActionType: Octopus.Script
+      Notes: 'Always runs'
+      IsDisabled: false
+      CanBeUsedForProjectVersioning: true
+      IsRequired: false
+      WorkerPoolVariable: WORKER_POOL
+      Condition: Success
+      Properties:
+        Octopus.Action.RunOnServer: 'true'
+        Octopus.Action.EnabledFeatures: Octopus.Features.JsonConfigurationVariables,Octopus.Features.SubstituteInFiles
+        Octopus.Action.Script.ScriptSource: Package
+        Octopus.Action.Script.ScriptFileName: deploy.ps1
+        Octopus.Action.Package.PackageId: "#{Octopus.Project.Name}"
```

Comment:
Is this required? I don't want the devs to worry about this.

Reply:
Incase they want to use bash/python script or want to use Inline Script.. Or do you only want them to use Poweshell?

Reply:
I'm specifically asking about `Octopus.Action.Package.PackageId` which won't change ever.